### PR TITLE
updating upgrade_status: completed

### PR DESCRIPTION
**Pull request by juliaElastic**
## What is the problem this PR solves?

https://github.com/elastic/kibana/issues/139704

Updating `upgrade_status: completed` agent field when an upgrade is acked, this is a fix instead of kibana relying on `upgraded_at: null` field, which caused a Lucene bug.

## How does this PR solve the problem?

The `upgrade_status` is set to `started` initially by kibana, when the action is started. Fleet Server then has to set it to `completed`, to signal to kibana that the action has completed.

## How to test this PR locally

1. Enroll fleet server, and many agents with horde
2. Bulk select agents and upgrade immediately
3. Wait for upgrade to complete, notice that agents are gradually showing up as `Healthy` and the Upgrade callout decreases in count.
4. Expect that the lucene error does not come anymore on the `agents` API call

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: internal/pkg/model/schema.go [hunk 1]
```diff
@@ -190,6 +190,9 @@ type Agent struct {
 	// Date/time the Elastic Agent was last upgraded
 	UpgradedAt string `json:"upgraded_at,omitempty"`
 
+	// Upgrade status
+	UpgradeStatus string `json:"upgrade_status,omitempty"`
```
Comment (michel-laterman) [inline internal/pkg/model/schema.go:194]: This file is generated from `model/schema.json`; you should update that and run the json schema generator in order to update the go definition.

Comment (juliaElastic) [reply internal/pkg/model/schema.go:194]: Do you know how can I run the generator?

**Comment by michel-laterman** [comment]
I think we would also need to add the new attribute to the ES schema definition: https://github.com/michel-laterman/elasticsearch/blob/master/x-pack/plugin/core/src/main/resources/fleet-agents.json

I have an example PR where I added fields to the actions schema: https://github.com/elastic/elasticsearch/pull/86167

File: internal/pkg/model/schema.go (hunk 1 above)
Comment (juliaElastic) [reply internal/pkg/model/schema.go:187]: updated

**Comment by juliaElastic** [comment]
@michel-laterman Do you know how to fix this build error?
```
14:44:12  Makefile:116: recipe for target 'check-no-changes' failed
14:44:12  make[1]: *** [check-no-changes] Error 1
14:44:12  make[1]: Leaving directory '/var/lib/jenkins/workspace/_server_fleet-server-mbp_PR-1833/src/github.com/elastic/fleet-server'
14:44:12  Makefile:74: recipe for target 'check-ci' failed
14:44:12  make: *** [check-ci] Error 2
```

**Comment by juliaElastic** [comment]
@michel-laterman are you ready to approve?

**Review by aleksmaus — APPROVED** [review]

**Pull request merged**